### ISSUE TITLE
Can't establish a connection to wss://tracker.webtorrent.io

### ISSUE THREAD

**Issue by andreipoe** [kern/filepizza#68]
Hello,

I can't manage to transfer files, either from the public instance or running from my own server, using both Firefox and Chrome. If running the Docker package, the error says:

```
WebSocket connection to 'wss://tracker.webtorrent.io/' failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVED @js:43
```

On https://file.pizza, the error message is:

```
ICE failed, add a TURN server
```

According to [this webtorrent issue](https://github.com/webtorrent/webtorrent-desktop/issues/1017), `tracker.webtorrent.io` no longer exists. Perhaps this needs to be replaced in File Pizza?

Thanks.

**Comment by adeekshith** [comment]
@kern Is this related to the same issue you mentioned on HN? https://news.ycombinator.com/item?id=14923391

I tried file pizza a few times last week but it never worked. Will be glad this gets fixed soon. Thanks :)

**Comment by kern** [comment]
This issue doesn't seem to be related to the slow connection start times. :(

I've fixed this in f1952c3, however! Let me know if you're still running into issues.
